### PR TITLE
New style for demo, updated readme and one bug

### PR DESCRIPTION
**Pull request by jreklund**
Hi, I have given the demo a face lift and therefore found a bug in the process.

Readme.md
- Spelling of cURL
- Removed shortcode array, not compatible with PHP 5.3
- SSL layout broken after Github update
- Added troubleshooting for languages

demo/cache.php
- HTML/CSS cleanup

demo/inc.php
- Escape single and double quotes, with UTF-8
  - PHP 5.3 will use ISO-8859-1	as default
  
demo/index.html
- HTML/CSS cleanup

demo/movie.php
- HTML/CSS cleanup
- PHP errors regarding "Filming Locations"
- PHP regular expression for movie.php/person.php have changed

demo/person.php
- HTML/CSS cleanup
- PHP regular expression for movie.php/person.php have changed

demo/search.php
- HTML/CSS cleanup
- You can now copy tt0834001 or nm0851281 directly into "or IMDb ID" - no need to remove tt|nm

demo/style.css
- CSS stylesheet

src/Imdb/Title.php
- If you visit a "TV Episode" episodes() will be stuck in a infinity loop, reason unknown

### CODE REVIEW

File: src/Imdb/Title.php [hunk 1]
```diff
@@ -1607,7 +1607,7 @@ public function episodes() {
       if (!$this->seasons()) {
         $ser = $this->get_episode_details();
         if (isset($ser['imdbid'])) {
-          $show = new Title($ser['imdbid'], $this->config);
```
Comment (tboothman) [inline src/Imdb/Title.php]: Can you undo this change please, this might solve one problem but it'll create another

**Comment by tboothman** [comment]
Thanks for this, it looks a lot nicer.
If you revert 905e352 I'll merge it in and fix that problem properly myself

File: src/Imdb/Title.php (hunk 1 above)
Comment (jreklund) [inline src/Imdb/Title.php]: I have now revered this.
